registry/flutter_cocoon.test

```diff
@@ -0,0 +1,8 @@
+contact=flutter-infra@google.com
+fetch=git clone https://github.com/flutter/cocoon.git tests
+fetch=git -C tests checkout 5dc667795520265c7796050fa2365e262276ed1a
```

Comment:
Does the commit need to be updated to the Cocoon tip of tree?

Reply:
Yeah, you're right. I updated the hash.